modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,38 @@ Containers are available for the following CPU architectures:
 * You are logged in as an administrator on the {ocp-short} web console.
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the Operator resources through role bindings or cluster role bindings.
+====
 
-. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* > *OperatorHub*.
+.Procedure
 
+. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators* -> *OperatorHub*.
```

Comment:
```suggestion
. In the *Administrator* perspective of the {ocp-short} web console, navigate to *Operators > OperatorHub*.
```
IBM guidance is to use the greater than symbol. Looks like it's also supposed to be bold.
https://www.ibm.com/docs/en/ibm-style?topic=elements-menus-navigation#navigation-trees